File: src/main/java/de/burrotinto/burroPlayer/interfaces/serial/executors/MVCPlayerExecutor.java
package de.burrotinto.burroPlayer.interfaces.serial.executors;

import de.burrotinto.burroPlayer.core.media.remote.IndexMediaRemoteService;
import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Service;

/**
 * Created by derduke on 16.02.17.
 */
@Service
@Slf4j
@RequiredArgsConstructor
public class MVCPlayerExecutor implements PlayerExecutor {
    private final IndexMediaRemoteService indexMediaRemoteService;

    @Override
    public void execute(int command) {
        log.info("Videostartbefehl: " + command);
        indexMediaRemoteService.play(command);
    }
}

File: src/main/kotlin/de/burrotinto/burroplayer/interfaces/serial/executors/MVCPlayerExecutor.kt
package de.burrotinto.burroplayer.interfaces.serial.executors

import de.burrotinto.burroplayer.core.media.remote.IndexMediaRemoteService
import org.springframework.stereotype.Service

/**
 * Created by derduke on 16.02.17.
 */
@Service
class MVCPlayerExecutor(private val indexMediaRemoteService: IndexMediaRemoteService) : PlayerExecutor {

    override fun execute(command: Int) {
        indexMediaRemoteService.play(command)
    }
}
